ci/checks/ensure-version-bumped.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+exit_code=0
+
+last_tag=$(git describe --tags --abbrev=0)
+last_tag_version=${last_tag:1}
```

Comment:
Nit, more self-explanatory:

```suggestion
last_tag_version=${last_tag/v/}
```